chart/values.schema.json

```
@@ -3764,6 +3764,19 @@
                             }
                         }
                     }
+                },
+                "securityContext": {
+                    "description": "Security context for the cleanup job pod. If not set, the values from `securityContext` will be used.",
+                    "type": "object",
+                    "$ref": "#/definitions/io.k8s.api.core.v1.PodSecurityContext",
+                    "default": {},
+                    "examples": [
+                        {
+                            "runAsUser": 50000,
```

```suggestion
                            "runAsUser": 999,
```

This will be a better example, as this is the uid of the `redis` user.